src/CommandingOfficers/AW3/AW3ValueFinders.java

```diff
@@ -0,0 +1,73 @@
+package CommandingOfficers.AW3;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.IValueFinder;
+import Engine.UnitActionFactory;
+import Units.Unit;
+import Units.UnitModel;
+
+public abstract class AW3ValueFinders
+{
+  public static class CostValueFinder implements IValueFinder
+  {
+    int maxDamage = 3;
+    @Override
+    public int getValue(Commander attacker, Unit unit)
+    {
+      // Ignore Oozium, if we ever add those
+      int health = unit.getHealth();
+      if( health < 10 )
+        return 2;
```

Comment:
Value is always 2 if health is less than 10? Whereas full-health units start at 1000? Is this right?

Reply:
Don't ask me, I just work here.
It's from Xenesis's meteor datamine thread.

Reply:
Drop a short explanation and link in a comment, maybe? That's super non-obvious at best, and looks like a bug on a normal day.

Reply:
Done.

Reply:
And you didn't share your newfound knowledge back with the rest of humanity? for shame...

Reply:
I did tell Xenesis. What she puts in her thread is her prerogative, lol
I could make my own account, ooooor I could be lazy.

At any rate, this code is open sauce.